products/sle-micro/main.pm

```diff
@@ -43,4 +60,7 @@ if (is_updates_test_repo && !get_var('MAINT_TEST_REPO')) {
 
 return 1 if load_yaml_schedule;
 
+load_boot_from_disk_tests();
```

Comment:
All tests use yaml schedule for now, but just to avoid future regressions if something else moves to main.pm, can you add a condition here so that containers are not there by default?
```
if (is_container_test) {
    load_boot_from_disk_tests();
    oad_host_tests_podman();
}
```

Reply:
good point! Updated